jib-core/src/main/java/com/google/cloud/tools/jib/builder/BuildConfiguration.java

```diff
@@ -49,6 +49,9 @@
     /** Known registry credentials to fallback on. */
     KNOWN_REGISTRY_CREDENTIALS(false),
 
+    /** Tell the layer build to use reproducibility features. */
+    REPRODUCIBLE_BUILDS(false),
```

Comment:
nit: Perhaps `ENABLE_REPRODUCIBLE_BUILDS` (and the getter/setter) to keep it consistent with the usage in `LayerBuilder`?